src/test/scala/ru/yandex/qatools/allure/scalatest/AllureReporterSpec.scala

```diff
@@ -33,9 +35,9 @@ class AllureReporterSpec extends FlatSpec with BeforeAndAfter {
     allure = mock(classOf[Allure])
     reporter = spy(new AllureReporter)
     reporter.setLifecycle(allure)
-    doReturn(testUuid).when(reporter).getSuiteUuid(any(classOf[String]))
```

Comment:
The culprits are the two `doReturn`s in the [Mockito](http://static.javadoc.io/org.mockito/mockito-core/2.4.5/org/mockito/Mockito.html#method_summary) object, where the compiler cannot choose between "empty varargs" and "no varargs". [This SO answer](http://stackoverflow.com/questions/13358705/force-single-argument-in-scala-varargs) suggests an alternative (and references a relevant SI ticket) by forcing varargs via `Nil: _*`, but the `when()` form appears to work just as well in this case.

Error for reference:

```
[error] /.../src/test/scala/ru/yandex/qatools/allure/scalatest/AllureReporterSpec.scala:39: ambiguous reference to overloaded definition,
[error] both method doReturn in object Mockito of type (x$1: Any, x$2: Object*)org.mockito.stubbing.Stubber
[error] and  method doReturn in object Mockito of type (x$1: Any)org.mockito.stubbing.Stubber
[error] match argument types (String)
[error]     doReturn(testUuid).when(reporter).getSuiteUuid(any(classOf[String]))
[error]     ^
[error] one error found
```